spec/Records.tex

```diff
@@ -219,15 +213,12 @@ \subsection{Record Initialization}
 var idCounter = 0;
 record UniqueID {
   var id: int;
-  proc UniqueID() { idCounter += 1; id = idCounter; }
+  proc init() { idCounter += 1; id = idCounter; }
 }
 
 writeln(new UniqueID());  // create and use a record value without a variable
 writeln(new UniqueID());
```

Comment:
This feels like a lost opportunity to demonstrate that `var tmp:UniqueID;` will call the `init()`.